Apps/WebClient/src/appsettings.Development.json

```diff
@@ -43,11 +43,13 @@
     },
     "ServiceEndpoints": {
         "ClinicalDocument": "http://localhost:3006/",
-        "Encounter": "http://localhost:3005/",
```

Comment:
Wouldn't that cause a bit of a headache running and debugging the encounter service locally?